documentation/modules/ref-list-of-kafka-cluster-resources.adoc

```diff
@@ -7,7 +7,31 @@
 
 The following resources will created by the Cluster Operator in the Kubernetes cluster:
 
+.Shared resources
+
+`_cluster-name_-cluster-ca`:: Secret with the Cluster CA used to encrypt the cluster communication.
+`_cluster-name_-cluster-ca-cert`:: Secret with the Cluster CA public key. This key can be used to verify the identity of the Kafka brokers.
+`_cluster-name_-clients-ca`::  Secret with the Clients CA used to encrypt the communication between Kafka brokers and Kafka clients.
+`_cluster-name_-clients-ca-cert`:: Secret with the Clients CA public key. This key can be used to verify the identity of the Kafka brokers.
+`_cluster-name_-cluster-operator-certs`:: Secret with Cluster operators keys for communication with Kafka and ZooKeeper.
+
+.Zookeeper nodes
+
+`_cluster-name_-zookeeper`:: StatefulSet which is in charge of managing the ZooKeeper node pods.
+`_cluster-name_-zookeeper-_idx_`:: Pods created by the Zookeeper StatefulSet.
+`_cluster-name_-zookeeper-nodes`:: Service needed to have DNS resolve the ZooKeeper pods IP addresses directly.
+`_cluster-name_-zookeeper-client`:: Service used by Kafka brokers to connect to ZooKeeper nodes as clients.
+`_cluster-name_-zookeeper-config`:: ConfigMap which contains the ZooKeeper ancillary configuration and is mounted as a volume by the ZooKeeper node pods.
+`_cluster-name_-zookeeper-nodes`:: Secret with ZooKeeper node keys.
```

Comment:
it's strange having two entries with the same name here but then they are different in terms of type (Secret vs Service) but you understand that in the description. I am not saying that we should do it or right now, but maybe having a table with "resource name", "kind" and "description" could be better? WDYT?

Reply:
I had a thought about the table. But I do not think it works so easily ... if you have separate column for Kind, the descriptions get a bit weird ... either they duplicate the kind in their current form ... or they are hard to read on their own, e.g. `Zookeeper node keys`.

Reply:
Like the groupings, and I also like the description list format over table.

Reply:
@scholzj let me don't agree on that :-) For each row you have the resource name, the kind and description so something like:

`_cluster-name_-zookeeper-nodes`      Service     used to have DNS resolve the ZooKeeper pods IP addresses 
`_cluster-name_-zookeeper-nodes`      Secret       contains ZooKeeper node keys

You can think about the current state as a table missing the column separation ... see this (using |||| to separate)

`_cluster-name_-zookeeper-nodes` |||| Service |||| needed to have DNS resolve the ZooKeeper pods IP addresses directly.
`_cluster-name_-zookeeper-nodes`|||| Secret |||| with ZooKeeper node keys.

It's just that the kind is lost in the description. The table is much more readable

Reply:
That will not look good ... you have short kinds (Secret, Service) and long ones (Service Account, Pod Disruption Budget). The long ones will make the shorts ones in the same table hard to read.

Reply:
I don't think so, it happens in a lot of tables out there and I don't really think it's an issue. For sure the current format is harder to read compared to a table.

Reply:
I haven't seen this anywhere. This looks terrible ... `Secret                          with ZooKeeper node keys.`.

Reply:
If you insist on a table, I can do it with having the kind twice ... once in its row and once in the description. That looks stupid but is still much more readable.

Reply:
I have just suggested the table because the current way is not so readable to me.
Let's see what other @strimzi/maintainers think.

Reply:
@ppatierno It looks like nobody else cares about this. So how do you want to proceed?

Reply:
Maybe the "maintainers" alias got lost, let's try with direct @tombentley @samuel-hawker @Frawless :)

Reply:
Personally I think it is marginally easier to maintain without a table
I guess the question is how many users care about this list and of those who do, who would benefit from a table?
Those that care about specifics are more likely to be able to understand the information presented already in my opinion.